src/store/index.js

```diff
@@ -28,11 +29,6 @@ export default new Vuex.Store({
     tokenInfo: {},
     tokenBalances: [],
     isHiddenContent: true,
-    useSdkWallet: false,
-    useIframeWallet: false,
-    sdk: null,
-    isBackendLive: true,
-    cookiesConsent: { },
```

Comment:
I think this is not heavily related to aeternity module, proposing to leave it here